jobserver/api/jobs.py

```diff
@@ -190,7 +191,7 @@ def post(self, request, *args, **kwargs):
         # store backend state sent up from job-runner.  We might rename the
         # header this is passed in at some point but now this is good enough.
         if flags := request.headers.get("Flags", {}):
```

Comment:
The fallback here shouldn't be an empty dict, as I think that will cause an error for json.loads?

Reply:
Good catch! Fixed and rebased.